<!-- Use this launch file to implement the VFH algorithm in a pre-mapped environment using
costmap.launch -->

<launch>

<!-- Launch move_base, AMCL and teleop nodes -->
<include file = "$(find vector_field_histogram)/launch/costmap.launch"/>

<!-- Load VFH parameters -->
<rosparam file = "$(find vector_field_histogram)/param/vfhwp.yaml"/>

<!-- Launch vfh node -->
<!-- <node name="vfhROS" pkg="vector_field_histogram" type="vfhROS.py" output="screen"/> -->

<node name="vfhROSDWA" pkg="vector_field_histogram" type="vfhROSDWA.py" output="screen"/>

</launch>